utils/rpms/daos.spec

```
@@ -584,6 +583,10 @@ getent passwd daos_agent >/dev/null || useradd -s /sbin/nologin -r -g daos_agent
 # No files in a shim package
 
 %changelog
+* Fri Feb 02 2024 Ashley M. Pittman <ashley.m.pittman@intel.com> 2.3.103-15
+- Update pydaos install process
+- Add a dependency from daos-client-tests to daos-devel and gdb
```

```suggestion
- Add a dependency from daos-client-tests to daos-devel
```

since `gdb` shouldn't really be added.  The reason you found it necessary was due to a bug that needs fixing.

Maybe freshen up the date/time stamps here and in `debian/changelog` if you have to push again.

---

This changelog entry needs to be updated to reflect 9c20aca90d4ec71f907e3aa4985f3b557fc0ddca.